interface/main/calendar/add_edit_event.php

```diff
@@ -151,10 +151,17 @@
     $collectthis = json_sanitize($collectthis[array_keys($collectthis)[0]]["rules"]);
 }
 
-$group_disabled = ($_GET['group'] && !$g_edit && $have_group_global_enabled ) ? ' disabled=true; ' : '';
-if ($group_disabled) {
-    echo '<script>$(function () {$("input").prop("disabled", true);$("select").prop("disabled", true);}) </script>';
-}
+
+$group_disabled = ($_GET['group'] && !$g_edit && $have_group_global_enabled);
+?>
+<script type="text/javascript">
+    var phpData = {
```

Comment:
This is ok, but it'd be nicer if you called a function defined in add_edit_event.js so there's an explicit dependency without people having to hunt for it.  It follows the principle of element of least surprise.

Reply:
ie something like
```
if (window.AddEditEventController.init) {
AddEditEventController.init({
        groupDisabled: <?php echo $group_disabled ? 'true' : 'false'; ?>,
        validationRules: <?php echo json_encode($collectthis); ?>
});
``` 

Then you could define AddEditEventController in your add_edit_event.js
```
window.AddEditEventController = {
  init: function(phpData) { /* do code here */ }
};
```

Or you could do it as IIFE (Immediately Invoked Function Expression) to prevent data leakage
```
(function(window) { 
function init(phpData) { /* do code here */ }

window.AddEditEventController = init;
})(window);
```